SPEC.md

```diff
@@ -438,6 +465,68 @@ IPAM plugin examples:
  - Routes are expected to be added with a 0 metric.
  - A default route may be specified via "0.0.0.0/0". Since another network might have already configured the default route, the CNI plugin should be prepared to skip over its default route definition.
 
+### Well-known Structures
+
+#### IP
+
+```
+  "ips": [
+      {
+          "version": "<4-or-6>",
+          "address": "<ip-and-prefix-in-CIDR>",
+          "gateway": "<ip-address-of-the-gateway>",      (optional)
+          "interface": "<index into 'interfaces' list>" (not required for IPAM plugins)
```

Comment:
Again, I think this numeric index for `interface` shouldn't have quotes around the value.